phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/ViewSettings.java

```diff
@@ -250,7 +251,17 @@ private void initAboutPane() {
         gridPane.add(new TextWithStyle(translate("Git Revision:"), CAPTION_TITLE_CSS_CLASS), 0, 2);
         Hyperlink gitRevisionLink = new Hyperlink(applicationGitRevision);
         gitRevisionLink.setOnAction(event ->
-                hostServicesSupplier.get().showDocument("https://github.com/PlayOnLinux/POL-POM-5/commit/" + applicationGitRevision)
+                {
+                    String url = "https://github.com/PlayOnLinux/POL-POM-5/commit/" + applicationGitRevision;
+                    // use xdg-open until HostServices works
+                    Runtime runtime = Runtime.getRuntime();
+                    try {
+                        runtime.exec("xdg-open " + url);
```

Comment:
You mean, because OpenJDK 8 often doesn't include the JavaFX classes? Sry, I forgot that...
Is it possible/ok for us to use awt classes? If we can use awt classes, then maybe the following code could be a solution?
```java
import java.awt.Desktop;
import java.net.URI;

// ...

if(Desktop.isDesktopSupported())
{
  Desktop.getDesktop().browse(new URI("http://www.example.com"));
}
```
(see also [here](http://stackoverflow.com/questions/5226212/how-to-open-the-default-webbrowser-using-java))